classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- [ ] The student array should be accessible, kindly consider adding `attr_reader :student` to your class so it can be accessible